test/Interop/Cxx/stdlib/use-std-hash-typechecker.swift

```diff
@@ -0,0 +1,10 @@
+// RUN: %target-typecheck-verify-swift %s -I %S/Inputs -cxx-interoperability-mode=default
```

Comment:
How does this pass on Linux? On macOS it errors with:
```
[2025-12-02T05:32:47.345Z] <unknown>:0: error: duplicate input file '/Users/ec2-user/jenkins/workspace/swift-PR-macos-smoke-test/branch-main/swift/test/Interop/Cxx/stdlib/use-std-hash-typechecker.swift'
```
since `%target-typecheck-verify-swift` includes `%s`